tests/test_reader.py

```diff
@@ -0,0 +1,135 @@
+import pytest
+from PIL import Image
+
+from readlif.reader import LifFile
+
+
+def test_lif_file_with_file_path(valid_lif_filepath):
+    lif_image = LifFile(valid_lif_filepath).get_image(0)
+    lif_image_frame = lif_image.get_frame(z=0, t=0, c=0)
+    assert lif_image_frame
```

Comment:
To me, asserting an object means the object could be None or False. But I suspect that's not the case here. How about

```suggestion
    # Runs without error
    lif_image.get_frame(z=0, t=0, c=0)
```

Reply:
Yeah, agreed. I don't know what I was thinking here.